### DISCUSSION TITLE
[popover] Light dismiss doesn't work on iOS/iPadOS

### DISCUSSION THREAD

**Discussion by Toitv** [tailwindlabs/tailwindcss#15515] [Help]
The click/touch on the backdrop of a open [popover], does not close the [popover] on Safari iOS/iPadOS. It does close the [popover] on the Safari macOS, Chrome desktop, Firefox desktop (with a polyfill), Chrome Android, Firefox Android (with a polyfill). 

The [popover] should close when clicked/touched on the backdrop [popover]::backdrop or pressing the escape key on a keyboard.

**Comment by wongjn** [comment]
Do you mean the [HTML Popover API](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/popover)? If so, could you explain how Tailwind influences it's behavior vs normal behavior please? I don't believe Tailwind should affect it, but then again, I haven't used it.
